src/libraries/System.Collections.Immutable/tests/ImmutableArrayTest.cs

```diff
@@ -1631,10 +1631,11 @@ public void CopyToInvalid(IEnumerable<int> source)
             // ImmutableArray<T>.CopyTo defers to Array.Copy for argument validation, so
             // the parameter names here come from Array.Copy.
 
-            AssertExtensions.Throws<ArgumentNullException>("destinationArray", "dest", () => array.CopyTo(null));
-            AssertExtensions.Throws<ArgumentNullException>("destinationArray", "dest", () => array.CopyTo(null, 0));
-            AssertExtensions.Throws<ArgumentNullException>("destinationArray", "dest", () => array.CopyTo(0, null, 0, 0));
-            AssertExtensions.Throws<ArgumentNullException>("destinationArray", "dest", () => array.CopyTo(-1, null, -1, -1)); // The destination should be validated first.
+            // Skip checking .NETFramework parameter name is either 'destinationArray' or 'dest'
+            AssertExtensions.Throws<ArgumentNullException>("destinationArray", netFxParamName: null, () => array.CopyTo(null));
```

Comment:
```suggestion
            AssertExtensions.Throws<ArgumentNullException>("destinationArray", "dest", () => array.CopyTo(null));
```
This is the style used in all other places that compensate for this problem. Also, the comment is unnecessary - none of the other places have it.